pkg/services/alerting/alert_notification_models.go

```diff
@@ -1,4 +1,4 @@
-package models
```

Comment:
not a blocker but I would suggest moving it to `services/alerting/alerts` (and possibly rename the package to something more generic (eg `services/alerting/models`) unless this was on purpose in order to resolve any cycle dependencies.

Reply:
There are already other alerting-related models under `services/alerting`; the only ones I moved to `services/alerting/alerts` were the models that needed to be separate. 

I like `services/alerting/models`, I'll make that change, but are you sure about moving this file as well (and not the other models that were already in there)? I don't feel that strongly either way, but the other commands and similar types are under `alerting` and I thought it was best to keep the models in that package if possible 🤷

Reply:
@yuri-tceretian I did move these to `services/alerting/models`; this PR is because we are removing everything from the `models` package in case that's the `models` package you're asking about.

Reply:
sorry for not being clear, I meant `services/alerting/models`. Currently, it is in `services/alerting`.

Reply:
Gotchya - yes, I will move these also 👍